video.qmd

```diff
@@ -0,0 +1,3 @@
+---
+  title: "Video"
+---
```

Comment:
maybe rename to `Videos` (plural), also the file.

otherwise it is good to have these placeholder pages - when they are empty I would suggest to have a one line "To be added later." on each of them